sdk/textanalytics/ai-text-analytics/src/lro/analyze/operation.ts

```diff
@@ -320,13 +322,13 @@ export class BeginAnalyzeActionsPollerOperation extends AnalysisPollOperation<
       tracingOptions: this.options.tracingOptions
     });
 
-    state.createdOn = operationStatus.operationMetdata?.createdOn;
-    state.expiresOn = operationStatus.operationMetdata?.expiresOn;
-    state.lastModifiedOn = operationStatus.operationMetdata?.lastModifiedOn;
-    state.status = operationStatus.operationMetdata?.status;
-    state.actionsSucceededCount = operationStatus.operationMetdata?.actionsSucceededCount;
-    state.actionsFailedCount = operationStatus.operationMetdata?.actionsFailedCount;
-    state.actionsInProgressCount = operationStatus.operationMetdata?.actionsInProgressCount;
+    state.createdOn = operationStatus.operationMetdata.createdOn;
+    state.expiresOn = operationStatus.operationMetdata.expiresOn;
```

Comment:
this one can still be null right? should the check remain?

Reply:
same comment for health lro

Reply:
what check? the `?`? 

the `?` applies to `operationMetdata` itself which is now required. You're correct that `expiresOn` is optional but that is fine because `state.expiresOn` is also optional.